astacus/coordinator/api.py

```diff
@@ -114,22 +120,30 @@ async def _list_backups(*, req: ipc.ListRequest = ipc.ListRequest(), c: Coordina
     return list_response
 
 
+@router.get("/delta/list")
+async def _list_delta_backups(*, req: ipc.ListRequest = ipc.ListRequest(), c: Coordinator = Depends(), request: Request):
+    # This is not supposed to be called very often, no caching necessary
+    return await to_thread(list_delta_backups, req=req, json_mstorage=c.json_mstorage)
+
+
 @router.post("/cleanup")
 async def cleanup(*, op: CleanupOp = Depends(CleanupOp.create), c: Coordinator = Depends()):
     runner = await op.acquire_cluster_lock()
     return c.start_op(op_name=OpName.cleanup, op=op, fun=runner)
 
 
 @router.get("/{op_name}/{op_id}")
+@router.get("/delta/{op_name}/{op_id}")
```

Comment:
because operation status url depends on the request path, therefore status urls for `/delta/X` look like `/delta/X/N`
```
    def start_op(self, *, op: Op, op_name: str, fun: Callable[[], Any]) -> Op.StartResult:
        ...
        url = self.request_url
        status_url = urlunsplit((url.scheme, url.netloc, f"{url.path}/{op.op_id}", "", ""))
```